packages/front-end/pages/metrics.tsx

```diff
@@ -247,13 +297,21 @@ const MetricsPage = (): React.ReactElement => {
             >
               <td>
                 <Link href={`/metric/${metric.id}`}>
-                  <a
-                    className={`${
-                      metric.status === "archived" ? "text-muted" : "text-dark"
-                    } font-weight-bold`}
+                  <Tooltip
+                    tipPosition="top"
+                    tipMinWidth="240px"
+                    ToolTipComp={<MetricToolTipComp metric={metric} />}
                   >
```

Comment:
Instead of adding the tooltip to the metrics page, add it to the experiment results table.  Code for that is here: https://github.com/growthbook/growthbook/blob/main/packages/front-end/components/Experiment/CompactResults.tsx#L104